.bit/responses/2.7-Week 2 Step 7.md

```diff
@@ -19,22 +19,27 @@ Week 2 Step 7 ⬤⬤⬤⬤⬤⬤⬤◯ | 🕐 Estimated completion: 15-25 minute
 
 | Generation  | Range         |
 |-------------|---------------|
-| GenZ        | 5 < age < 25  |
-| GenY        | 24 < age < 41  |
-| GenX        | 40 < age < 57  |
-| BabyBoomers | 56 < age < 76  |
+| GenZ        | 5 ≤ age ≤ 25  |
+| GenY        | 26 ≤ age ≤ 41  |
+| GenX        | 42 ≤ age ≤ 57  |
+| BabyBoomers | 58 ≤ age ≤ 76  |
 | Unknown     | Any other age |
 
 - [ ] 🚀 Commit your updated code to `song4u/index.js` on the `song4u` branch
 
 ### 🚧 Test Your Work
-To test your work, try texting a **jpg** image to your Twilio number (with a face!). You should receive a text back that **only contains the generation.**
+To test your work, try texting a **jpg** image to your Twilio number (with a face!). 
```

Comment:
Also, add instructions to log the age and/or generation out in the console. That way, they can at least check that their function is successful. Ex: 

```
After you text the image, you can `context.log` the age and/or generation to...
```